plugins/parodos/src/components/ParodosPage/navigationMap.tsx

```diff
@@ -5,9 +5,10 @@ export const pluginRoutePrefix = '/parodos';
 
 export const navigationMap = [
   { label: 'Projects', routes: ['/project-overview'], icon: <ProjectsIcon /> },
+  { label: 'Workflows', routes: ['/workflows'], icon: <AssessmentIcon /> },
   {
     label: 'Assessment',
-    routes: ['/workflow', '/onboarding/'],
+    routes: ['/onboarding/'],
```

Comment:
removing that route breaks the Assessment link in the nav, should the link be removed?  I did not add these routes so I don't really know why they are an array and I'm pretty sure it is a bad idea


https://user-images.githubusercontent.com/118328/235981620-a35384f9-d09d-4087-9744-8d98c45a080b.mp4

Reply:
I've fixed the route in PluginRouter